src/python/WMCore/Storage/TrivialFileCatalog.py

```diff
@@ -196,55 +232,144 @@ def tfcFilename(contactString):
     """
     value = contactString.replace("trivialcatalog_file:", "")
     value = _TFCArgSplit.split(value)[0]
-    path = os.path.normpath(value)
+    path = os.path.normpath(value) #does this resolve relative and symbolic path?
     return path
 
-
-def readTFC(filename):
+def readTFC(filename,storageAttr=None,useTFC=True):
```

Comment:
As I explained above the default here is not important since the function is always called explicitly. Maybe I can remove the default value for clarity (I always weight the backward compatibility higher than the new codes so the useTFC=True reflex that)